src/ShowScore.js

```diff
@@ -0,0 +1,17 @@
+import { showData, table, apiURL } from './variable.js';
+
+showData.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  table.innerHTML = '';
+  async function showScore() {
+    const show = await fetch(apiURL);
```

Comment:
- [ ] It would be great if you use the arrow function **`const showScore = async () => {}`** instead of the 
**`async function showScore()`** to follow your project requirements :+1: